website/content/en/preview/getting-started/getting-started-with-eksctl/cloudformation.yaml

```
@@ -73,6 +73,10 @@ Resources:
             Action:
               - iam:PassRole
             Resource: !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/KarpenterNodeRole-${ClusterName}"
+          - Effect: Allow
+            Action:
+              - eks:DescribeCluster
+            Resource: !Sub "arn:${AWS::Partition}:eks:*:${AWS::AccountId}:cluster/${ClusterName}"
```

```suggestion
            Resource: !Sub "arn:${AWS::Partition}:eks::${AWS::AccountId}:cluster/${ClusterName}"
```

Is this ARN correct with the `*` in the middle?

---

Yes, we need the * because the cluster can be in any region.

---

Sure, agree!